### DISCUSSION TITLE
argocd:v2.12.1 Unable to create application:application spec for app isinvalid: InvalidSpecError:application destination server"https://kubernetes.default.svcand namespace 'cicd-test'donot match any of the allowed destinations in project test1234'

### DISCUSSION THREAD

**Discussion by hbhe0316** [argoproj/argo-cd#19919] [Q&A]
![2221](https://github.com/user-attachments/assets/bb8b35d9-2045-44bf-903e-d2bbbf0a29c0)

wrong issue:
Unable to create application:application spec for app isinvalid: InvalidSpecError:application destination server"https://kubernetes.default.svcand namespace 'cicd-test'donot match any of the alloweddestinations in projecttest1234'

argocd version:argocd:v2.12.1
when I create a app meet a issue.I don't know why.

**Comment by rumstead** [comment]
Your argo cd project called `test1234` doesn't allow your application to be deployed to the `in-cluster` cluster to namespace `cicd-test`. You will need to update it to allow it to deploy there.

https://argo-cd.readthedocs.io/en/stable/user-guide/projects/#managing-projects
